Core/include/Acts/Propagator/Navigator.hpp

```diff
@@ -714,7 +718,8 @@ class Navigator {
           } else {
             s = state.stepping.stepSize.min();
           }
-          opening_angle = std::atan((1 - std::cos(s * ir)) / std::sin(s * ir));
+          opening_angle = std::atan((1 - std::cos(s * ir)) / std::sin(s *
+        ir));
```

Comment:
who doesn't love clang format :)

Reply:
is this some kind of oscillating clang-format thing where it doesn't like it either way or something? we didn't change the version of clang-format, which would have required changing everything at once anyway, right?

Reply:
Should be gone